indicators/views/bulk_indicator_import_views.py

```diff
@@ -451,7 +452,7 @@ def post(self, request, *args, **kwargs):
         ProgramAuditLog.log_template_uploaded(request.user, program)
 
         wb = openpyxl.load_workbook(request.FILES['file'])
-        ws = wb.get_sheet_by_name(TEMPLATE_SHEET_NAME)
+        ws = wb.worksheets[0]
```

Comment:
Just thinking of an edge case, if a user for some reason adds other sheets to the template and uploads it, would that cause a problem?

Reply:
It would probably fail when trying to read the program, which I think is ok.

Reply:
I've been trying to avoid having to do reverse translations, which I think are doable, but a little weird.